src/rocm_docs/core.py

```diff
@@ -91,6 +91,14 @@ class _DefaultSettings:
     epub_show_urls = _ConfigDefault("footnote")
     exclude_patterns = _ConfigExtend(["_build", "Thumbs.db", ".DS_Store"])
     numfig = _ConfigDefault(True)
+    linkcheck_timeout = _ConfigDefault(10)
+    linkcheck_request_headers = _ConfigMerge(
+        {
+            r'https://docs.github.com/': {'User-Agent': 'Mozilla/5.0 (X11; Ubuntu; Linux i686; rv:112.0) Gecko/20100101 Firefox/112.0'}
+        }
+    )
+    linkcheck_anchors_ignore = _ConfigExtend([])
+    linkcheck_ignore = _ConfigExtend([])
```

Comment:
```suggestion
```
These don't do anything as is (extending a list with an empty list doesn't remove its content). For `lincheck_ignore` the default is to ignore nothing so this is the same, for `linkcheck_anchors_ignore` we can just leave the default (`^!`) as is, I don't think it causes any harm.